it/uk/gov/hmrc/agentmappingfrontend/controllers/MappingControllerISpec.scala

```diff
@@ -31,20 +31,50 @@ class MappingControllerISpec extends BaseControllerISpec with AuthStubs {
       val result = callEndpointWith(request)
       status(result) shouldBe 200
       checkHtmlResultContainsMsgs(result, "connectAgentServices.start.title")
-      bodyOf(result) should include("TARN-000-0001")
     }
 
-    "display the start page for unAuthenticated user" in {
+    "display the /start/sign-in-required for unAuthenticated or user without HMRC-AS-AGENT/ARN" in {
```

Comment:
This test is just for an unauthenticated user that's not even logged in isn't it?